### ISSUE TITLE
Please fix 

### ISSUE THREAD

**Issue by Bharat99978** [stackblitz/bolt.new#4060]
### Describe the bug

Please fix 

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-ui2ufb3t

### Steps to reproduce

Please fix problem

### Expected behavior

![Screenshot_20241212_102748](https://github.com/user-attachments/assets/83ea662c-f10c-4a15-8576-4d0091ce1e05)

### Screen Recording / Screenshot

![Screenshot_20241212_102748](https://github.com/user-attachments/assets/042051b2-53d9-4cc1-97a4-1f42d6d06e2c)

### Platform

```
Browser name = Chrome
Full version = 122.0.0.0
Major version = 122
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 45154662,
  "usedJSHeapSize": 38308250,
  "jsHeapSizeLimit": 1098907648
}
```

### Additional context

Error:
```
out-of-memory: RuntimeError: unreachable
    at wasm://wasm/a36bee7a:wasm-function[10]:0x1f9f
    at wasm://wasm/a36bee7a:wasm-function[7]:0x1e60
    at _0x43b5e9 (https://zp1v56uxy8rdx5ypatb0ockcb9tr6a-oci3.w-credentialless-staticblitz.com/filesystem-worker.fc837ba8.js:1:352094)
    at _0x5d803b (https://zp1v56uxy8rdx5ypatb0ockcb9tr6a-oci3.w-credentialless-staticblitz.com/filesystem-worker.fc837ba8.js:1:352762)
    at async _0x1104d3._initSlabAsync (https://zp1v56uxy8rdx5ypatb0ockcb9tr6a-oci3.w-credentialless-staticblitz.com/filesystem-worker.fc837ba8.js:1:355280)
    at async _0x1104d3.create (https://zp1v56uxy8rdx5ypatb0ockcb9tr6a-oci3.w-credentialless-staticblitz.com/filesystem-worker.fc837ba8.js:1:353217)
    at async _0x2d6cac.init (https://zp1v56uxy8rdx5ypatb0ockcb9tr6a-oci3.w-credentialless-staticblitz.com/filesystem-worker.fc837ba8.js:1:493966)
```

**Comment by jcmoore** [comment]
tl;dr: try loading bolt.new on a machine with more RAM or free up device memory

Not from the team at all but it looks like you are using bolt.new on a mobile phone.  The way bolt and stackblitz work (from my limited/outsider understanding) is that they rely on your machines memory to do much of anything.  If you are on a mobile phone, try working on a laptop or desktop instead.  If you are on a big capable machine, try cloading all your other tabs and apps to free up memory.

Best of luck!
